packages/core/src/EventManager.ts

```diff
@@ -1,5 +1,5 @@
 import { EventNormalizer, DomRangeDescription } from './EventNormalizer';
-import { LineBreakNode } from './VNodes/LineBreakNode';
+import { LineBreakNode } from '../../plugin-linebreak/src/VNodes/LineBreakNode';
```

Comment:
`EventManager` should not be aware of `LineBreakNode`, given that it's optional.